omod/src/test/java/org/openmrs/web/controller/ForgotPasswordFormControllerTest.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
```

Comment:
This file is where the problem is. If this starts detecting my data, this issue is solved as I will generate relevant tests and cleanup code ASAP.